docs/en/navigation.yml

```
@@ -729,16 +729,16 @@ nav:
               - Data Collection:
                   - Collection Management: insight/user-guide/collection-manag/collection-manag.md
                   - Service Monitor: insight/user-guide/collection-manag/service-monitor.md
-                  - Collection Metrics: insight/user-guide/collection-manag/metric-collect.md
-                  - insight-agent Status: insight/user-guide/collection-manag/agent-status.md
-                  - Custom Probe: insight/user-guide/collection-manag/probe-module.md
+                  - insight-agent Status: insight/user-guide/collection-manag/agent-status.md   
               - System Config:
                   - System Components: insight/user-guide/system-config/system-component.md
                   - System Config: insight/user-guide/system-config/system-config.md
                   - Modify Config: insight/user-guide/system-config/modify-config.md
               - Network Insight:
                   - Integrate DeepFlow: insight/best-practice/integration_deepflow.md
               - References:
+                  - Custom Probe: insight/user-guide/collection-manag/probe-module.md
+                  - Collection Metrics: insight/user-guide/collection-manag/metric-collect.md
```

```suggestion
                  - Insight Reference Metrics: insight/reference/uesd-metric-in-insight.md
                  - Custom Probe: insight/user-guide/collection-manag/probe-module.md
                  - Methods to Collect Metrics: insight/user-guide/collection-manag/metric-collect.md
```
`uesd-metric-in-insight.md` 这个文件名中 `uesd-`应该是`used-`，可以改一下

---

尚无英文文档，中文文档已重命名